### PR TITLE
feat(api): webhooks api

### PR DESCRIPTION
**Pull request by Xaviju**
https://taigaio.github.io/taiga-doc/dist/api.html#webhooks-list

### CODE REVIEW

File: src/app/api/webhooks/webhooks-api.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+
+  it('get webhook', () => {
+    const id = faker.random.number();
+
+    spectator.service.get(id).subscribe();
+    spectator.expectOne(`${ConfigServiceMock.apiUrl}/webhooks/${id}`, HttpMethod.GET);
+  });
+
+  it('patch', () => {
```
Comment (juanfran) [inline src/app/api/webhooks/webhooks-api.service.spec.ts:67]: put

Comment (Xaviju) [reply src/app/api/webhooks/webhooks-api.service.spec.ts]: fixed

File: src/app/api/webhooks/webhooks-logs-api.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { createHttpFactory, HttpMethod, SpectatorHttp } from '@ngneat/spectator';
+
+import { ConfigService } from '@/app/config.service';
+import { ConfigServiceMock } from '@/app/config.service.mock';
+import * as faker from 'faker';
+import { UtilsService } from '@/app/commons/utils/utils-service.service';
+import { WebhooksLogsApiService } from './webhooks-logs-api.service';
+
+
+describe('WebhooksApiService', () => {
```
Comment (juanfran) [inline src/app/api/webhooks/webhooks-logs-api.service.spec.ts:18]: WebhooksLogsApiService

Comment (Xaviju) [reply src/app/api/webhooks/webhooks-logs-api.service.spec.ts:18]: fixed!

File: src/app/api/webhooks/webhooks.model.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  key: string;
+  logs_counter: number;
+  name: string;
+  url: string;
+}
+
+export type WebhookCreationData = Pick<WebhookDetail, 'project' | 'name' | 'url' | 'key'>;
+
+export interface WebhookLog {
+  _error_message: string;
```
Comment (juanfran) [inline src/app/api/webhooks/webhooks.model.ts:21]: the interceptor will transform this to `errorMessage`

File: src/app/api/webhooks/webhooks-api.service.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    private http: HttpClient,
+    private config: ConfigService) { }
+
+  public get base() {
+    return `${this.config.apiUrl}/webhooks`;
+  }
+
+  public get webhooklogsBase() {
+    return `${this.config.apiUrl}/webhooklogs`;
+  }
```
Comment (bameda) [inline src/app/api/webhooks/webhooks-api.service.ts:29]: Maybe this get is not necessary

Comment (Xaviju) [reply src/app/api/webhooks/webhooks-api.service.ts]: fixed

File: src/app/api/webhooks/webhooks.model.ts (hunk 3 above)
Comment (Xaviju) [reply src/app/api/webhooks/webhooks.model.ts]: fixed

**Pull request merged**